wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -110,7 +107,7 @@ getHistoryLimited
     -> Maybe Word
     -> m ([CTx], Word)
 getHistoryLimited mCWalId mAccId mAddrId mSkip mLimit =
-    first applySkipLimit <$> getHistory mCWalId mAccId mAddrId
+    first applySkipLimit <$> getHistory mCWalId mAccId mAddrId (skip + limit)
```

Comment:
I don't like you process even transactions which you will skip, i.e. you will try to find addresses of transactions which you will skip in future. It's double work and it might reduce performance

Reply:
maybe it's enough for now, but I suggest to fix it